### ISSUE TITLE
innerGlitch - Science, Technology & Discovery District

### ISSUE THREAD

**Issue by BenDelGreco** [decentraland/districts#63]
**Title:** innerGlitch - Science, Technology & Discovery District

**Description:** innerGlitch is an educational project i’m actively working on since 3 months and it is dedicated to the spreading of science literacy and general curiosity using different kind of media. The idea is to open a segment of this project on Decentraland and adapt the base concepts of the innerGlitch project to VR education distributing revenue based on the contributions the members of the district community bring to the idea.

The Science, Technology & Discovery District will **not** be designed like a sort of virtual University, but it will be created a city-like layout with portions of LAND dedicated to particular topics, scientific conference rooms, interactive simulations, educational related activities, etc.

Imagine being able to experience how it is like to travel at the speed of light, imagine what would be like to stand on the edge of a black hole and see it with your eyes, imagine travel inside your arterial system.
innerGlitch - Science & Discovery District will be the place where to **_experience_** topics and concepts we are now able to **_only read._**

**Project Lead:** Ben Del Greco (theinnerglitch@gmail.com)

**Team:** It is me and my dog (who doesn’t contribute much) for now, so everybody else interested in the mission of this project is more than welcome. I am a Software developer and have some experience in 3D design, it would be nice to have contributions from someone with similar skills.

**Goal:**  The main goal of this District is to spread scientific knowledge through experience rather than the conventional and boring way. Some of the experiences may be monetized, others may be free. Ideally one or more areas of the District will be dedicated to public live forums where it will be possible to host public events.

**Category:** Public

**LAND Needed**: 5+ will make the minimum, but the ideal number is between 20 and 30 LAND. Above that, we’ll need a better district planner than me.

**Startup Costs:** No start-up costs are quantified yet. I hope in an open-source style collaboration (so no start-up costs) and in a after-things-are-done retribution.

**How will funds be raised:** We’ll see later in development if any funds are needed at all.

**How will contributors see a return value:** 
First - You’ll be co-founder of the first decentralized science-driven VR community. 
Second - Early contributors will have a better placement of their activities inside the district and early access to the planning of the social activities inside the district.
Third - LAND owner of the district will be able to monetize with rent of spaces, shows, events and science related games.  

**Attachments / Designs / Budgets:** Soon to come.

Anyone interested is welcome to comment or email me ideas and suggestions about the Science & Discovery District or your interest in participating in the building of this district :-)
This is an early stage project, some aspects may change after the request of interested community members.

**Comment by dclancy13** [comment]
Hi Ben! 
Are you in Slack? would like to send you a DM
Cheers!

**Comment by BenDelGreco** [comment]
@dclancy13  Yes, I'm on Slack, you can DM me at ben_del_greco, but it may be easier for me to answer at theinnerglitch@gmail.com 

Cheers!

**Comment by Denis17DCL** [comment]
I am already in DCL university. 
It might be better to cooperate?
Or you see some limitations or possible problems?

**Comment by BenDelGreco** [comment]
@Denis17DCL I love DCL University and I think it is a fantastic idea, but it is a lot different from innerGlitch and its mission.
In fact, in the Description you can read: "The Science, Technology & Discovery District will **not** be designed like a sort of virtual University, but it will be created a city-like layout...". This means that innerGlitch will not be a VR alterative to standard education, but it will be a VR City where simulations, games and experiences have the power of educational tool.
It will help using a new way of teaching through experiences rather than concepts and notions.

Anyhow, if you are interested, have ideas for the project or want to talk more about the district concept feel free to ask any questions here or via email.

Thank you!

**Comment by ghost** [comment]
@BenDelGreco if i get this right what you want to do is to create something educational, but with more practical stuff instead of theoretical like a university, right? if that's the case then i still think you should work closely together. the university could be a centered building and the innerGlitch could be a district around the university, so that if the subjects taught in class xyz in the university call for it, they can walk over to the practical testing site and try out stuff for themselves. i think i read something about the university being planned to be next to a forest, maybe you two could encircle it half-half with the university in the middle?

**Comment by BenDelGreco** [comment]
@KevinAT 
>"if i get this right what you want to do is to create something educational, but with more practical stuff instead of theoretical like a university, right?"

Right, teaching through experience rather than theoretical concepts is the mission for this district.

>"if that's the case then i still think you should work closely together."

I guess that is the best scenario, since it would be great to have a consistent theme while walking from parcel to parcel. Before making any decision though, I'm waiting around 2 weeks (starting from the date I published it) for the community feedback. Later, I'll move accordingly with what community members and potential collaborators have suggested.

>"the innerGlitch could be a district around the university, so that if the subjects taught in class xyz in the university call for it, they can walk over to the practical testing site and try out stuff for themselves."

I think this can be done for very few subjects. I'll give you two examples you can observe on Youtube to better explain this District idea.
There is an educational channel called "Khan Academy" which produces videos about algebra, chemistry, physics, etc...
Another educational channel is "Kurzgesagt - In a nutshell" which produces videos about every scientific topic the staff wants to discuss with the viewers.

The innerGlitch District wants to be more on the "Kurzgesagt" side in terms of subjects and way of teaching, while I see DCL University more on the "Khan Academy" side. (if there is something not clear with this comparison, let me know)

>" maybe you two could encircle it half-half with the university in the middle?"

I'm not sure about this, I should talk with DCL University team about this matter. 

___

Anyway, every contribution to this District concept is more than welcome. So let me know if you are interested in the project and/or in collaborating.

Thanks!

**Comment by EmmaMakhno** [comment]
If this district doesn't get enough land you can make it with my land in the A-Squat district

**Comment by BenDelGreco** [comment]
@EmmaMakhno Thanks, but I'm committing to this district because of the idea behind it. If the district doesn't get enough LAND (consider it is not listed yet), I'll prefer to find another way to bring to reality something similar to this project in Decentraland.

**Comment by Paradigm-Aethyrs** [comment]
I like this.  I'd like to know more about innerglitch

**Comment by BenDelGreco** [comment]
Hi @Paradigm-Aethyrs , we talk about it on rocket chat or via email. Here are my rc contact and email: @ben_del_greco and theinnerglitch@gmail.com

**Comment by MethaneRain84** [comment]
Hello everybody!

Super new and late to the game but I finally got some MANA dedicated to this district and I'm pretty excited for the possibilities!

Would things like physics and engineering simulations be an appropriate for what you are envisioning? I have a physics degree and trying to learn how to make simulations like University of Colorado's PHeT simulations.

I just downloaded rocket chat so I can move over there if desired.

**Comment by BenDelGreco** [comment]
Hi @MethaneRain84 ,
I'm really happy to have you on innerGlitch!

Contact me on RocketChat @ben_del_greco or send me your contact via email at: theinnerglitch@gmail.com
